tests/test_files/html/images.html

```diff
@@ -1,7 +1,3 @@
 <h1>Images should be removed</h1>
-<h2>Inline images</h2>
-<h2>Reference images</h2>
-<p>The reference is immediately after the image.</p>
-<p>The reference is not immediately after the image.</p>
-<p>This line separates the images from their references.</p>
-
+<p>text before  text after</p>
+<p>text before  text after</p>
```

Comment:
It is not clear what the purpose of this is. Please indicate intent for future devs.

Reply:
I believe this would be changed in `images.md`, something to the effect of
```
<p>no alt text string - text before  text after</p>
<p>alt text string - text before  text after</p>
```

Reply:
These are 2 instances of images being removed.  These are the output, rendered as HTML, of this markdown input:
```
text before ![alt text](https://www.example.com/test.jpeg) text after

text before ![alt text](https://www.example.com/test.webm "title text") text after
```
The addition causes the double white space.  Cleaning that up would require post processing.  I'm not inclined to do that because we don't list images as something we support.